.env.batistack

```
@@ -0,0 +1,68 @@
+APP_NAME="Batistack"
+APP_ENV=local
+APP_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_KEY key should go before the APP_NAME key

(UnorderedKey)